documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
 * Registered charities
 * Community organisations
 * Overseas organisations
+* Public bodies
 * Schools, universities and other educational establishments
 * Individuals
```

Comment:
- Religious bodies? 
(Those are not public bodies nor community organisations)

Reply:
As per comment above, voluntary and community organisations - it captures them all if we include the term "voluntary".